02_calculator/calculator.rb

```diff
@@ -1 +1,54 @@
-#write your code here
+def add(num1, num2)
```

Comment:
Good efforts here. I suggest finding Ruby methods that can help with each function. For example, the sum function can be a one line with `array.sum`, the sum method adds and returns total of all numbers. Then any edge cases can be added for preventing errors/issues.